chains/ethereum/listener.go

```diff
@@ -28,7 +28,7 @@ type Listener struct {
 	conn                 *Connection
 	subscriptions        map[EventSig]*ActiveSubscription
 	router               chains.Router
-	bridgeContract       *BridgeContract       // instance of bound bridge contract
+	bridgeContract       *BridgeContract       // instance of bound bridge bridgeContract
```

Comment:
tiny nitpick but can there be a bridge contract for something other than the bridge? Would it be clearer to say `instance of bound bridgeContract`.